requirements-dev.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with Python 3.10
+# This file is autogenerated by pip-compile with Python 3.9
```

Comment:
fyi for reviewer, we switched to py3.9 as primary dev env a while ago ([here](https://github.com/lvgig/tubular/pull/326), so this was picking up a leftover 3.10 ref. Seeing this led me to ctrl-shift-f for 3.10 where I found and updated another old ref

Reply:
thanks for the explainer